ui/src/embeddable/chart/ChartsComponent/index.jsx

```diff
@@ -266,6 +271,12 @@ const ChartComponent = ({ sources, data, type, title, subTitle, editing }) => {
       enableGridY = false;
       numberOfActiveBreeders();
       break;
+    case MARKET_CONCENTRATION_HHI:
+        useCropLegendsRow = false;
+        useFilterByCrops = false;
+        title = 'Market Concentration, as Measured by the HHI (Out of 10,000)';
```

Comment:
there is a config in the blocks component to put title and subtitle. Please remove this here, it will be configured from wp